app/controllers/application_controller.rb

```diff
@@ -4,6 +4,6 @@ class ApplicationController < ActionController::Base
   protected
 
   def configure_permitted_parameters
-    devise_parameter_sanitizer.permit(:sign_up, keys: [:name, :email, :password, :role, :is_confirmed, :company_id])
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:name, company_attributes: [:name, :domain]])
```

Comment:
change domain to subdomain and incorporate where change required

Reply:
done